i18n/de.po

```diff
@@ -19,7 +19,8 @@ msgstr ""
 "Freifunk-Knoten. </p><p>Bitte wähle zuerst einen Hostnamen nach dem Schema "
 "&lt;PLZ&gt;-&lt;Phantasiename&gt; (z.B. 54290-einhorn) aus, wobei PLZ die "
 "Postleitzahl an dem Standort des Knotens und Phantasiename ein von dir "
-"freigewählter Name ist.</p><p>Falls Du die Bandbreite für das Mesh-VPN "
+"freigewählter Name ist.</p><p>Achtung! Dieser Name könnte für immer öffentlich"
```

Comment:
I  would also leave the paragraph about the name-schema together with the hint and not put the hint in a new paragraph.
... so instead of "....&lt;/p&gt; &lt;p&gt;...." just do a "...&lt;br/&gt;..."